source/library/element-stub.js

```diff
@@ -0,0 +1,67 @@
+export default class ElementStub {
+	/**
+	 * Creates a new ElementStub instance. See {@link ElementStub#constructor}
+	 * @constructor
+	 * @private
+	 */
+	static create(...args) {
+		return new ElementStub(...args);
+	}
+
+	/**
+	 * Style proxy object
+	 * @type {Object}
+	 * @private
+	 */
+	style = {};
+
+	/**
+	 * Reference to currently active animationFrame
+	 * @type {Object}
+	 * @private
+	 */
+	frame = null;
+
+	/**
+	 * Creates a new ElementStub instance
+	 * @constructor
+	 * @param  {Node} element Node to stub
+	 * @param  {Window} [window=global.window] Global context to use
+	 * @return {ElementStub}         ElementStub instance
+	 * @private
+	 */
+	constructor(element, window = global.window) {
+		this.element = element;
+
+		const loop = () => {
+			this.render(this.element, this.style._style || this.style);
```

Comment:
The point of the style property is to serve as proxy for the "real" `HTMLElement.prototype.style`. It is exposed solely to `web-animations-js` (or any other WebAnimations poly-/ponyfill the user chooses) by binding the `element.animate` calls to instances of `HTMLElementStub` instead of the real HTMLElement. As such it is not exposed to the user in any way, alas no external API.